core/tests/webdriverio_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -411,8 +411,44 @@ describe('Core exploration functionality', function() {
       await explorationPlayerPage.expectExplorationToBeOver();
     });
 
+  it('should not show self-loop warning when navigating between ' +
+    'different states', async function() {
+    await explorationEditorMainTab.setContent(
+      await forms.toRichText('some content'));
+    await explorationEditorMainTab.setInteraction('NumericInput');
+    await explorationEditorMainTab.addResponse(
+      'NumericInput', async function(richTextEditor) {
+        await richTextEditor.appendBoldText('correct');
+      }, 'final card', true, 'IsInclusivelyBetween', -1, 3);
+    var responseEditor = await explorationEditorMainTab.getResponseEditor(0);
+    await responseEditor.markAsCorrect();
+    responseEditor = await explorationEditorMainTab.getResponseEditor(
```

Comment:
Suggest naming this defaultResponseEditor.

Reply:
done